docs/pages/tutorial/image-picker.md

```diff
@@ -87,6 +86,7 @@ Now we will take the data that we get from the image picker and use it to show t
 
 <!-- prettier-ignore -->
 ```js
+/* @info Import React to use useState */import React from 'react';/* @end */
```

Comment:
If you want to add imports to the second example, please list all of them.

Reply:
Done. I thought to remove it because in [a previous page](https://docs.expo.dev/tutorial/button) it is mentioned in the end to include only relevant code in the snippets.